client/src/components/ArrowButton.tsx

```diff
@@ -0,0 +1,19 @@
+import Arrow from "../assets/images/arrow.png"
+
+import "./ArrowButton.css";
+
+interface ArrowProps {
+  rotation: number // 0 points left, 90 points up, 180 points right, 270 points down
+  buttonFunc: (direction: string) => void
```

Comment:
think you could name this function something that makes it clearer what it's doing. one convention for event handlers is a name like handleClick or maybe even more explicitly, handleRotate or onRotate.